<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>109</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetPath" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="btHome">
        <property name="toolTip">
         <string>Go to user home directory</string>
        </property>
        <property name="statusTip">
         <string>Go to user home directory</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
          <normaloff>:/pktk/images/normal/home</normaloff>
          <disabledoff>:/pktk/images/disabled/home</disabledoff>:/pktk/images/normal/home</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btSavedViews">
        <property name="toolTip">
         <string>Views</string>
        </property>
        <property name="statusTip">
         <string>Saved views list</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
          <normaloff>:/pktk/images/normal/saved_views</normaloff>
          <disabledoff>:/pktk/images/disabled/saved_views</disabledoff>
          <disabledon>:/pktk/images/disabled/saved_views</disabledon>:/pktk/images/normal/saved_views</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btBookmark">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Bookmark</string>
        </property>
        <property name="statusTip">
         <string>Bookmark list</string>
        </property>
        <property name="whatsThis">
         <string>Bookmark</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
          <normaloff>:/pktk/images/normal/bookmarks</normaloff>
          <disabledoff>:/pktk/images/disabled/bookmarks</disabledoff>
          <disabledon>:/pktk/images/disabled/bookmarks_clear</disabledon>:/pktk/images/normal/bookmarks</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btLastDocuments">
        <property name="toolTip">
         <string>Last documents</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
          <normaloff>:/pktk/images/normal/saved_view_last</normaloff>
          <disabledoff>:/pktk/images/disabled/saved_view_last</disabledoff>:/pktk/images/normal/saved_view_last</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btHistory">
        <property name="toolTip">
         <string>History</string>
        </property>
        <property name="statusTip">
         <string>Last documents list</string>
        </property>
        <property name="text">
         <string>h</string>
        </property>
        <property name="icon">
         <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
          <normaloff>:/pktk/images/normal/history</normaloff>
          <disabledoff>:/pktk/images/disabled/history</disabledoff>
          <disabledon>:/pktk/images/disabled/history</disabledon>:/pktk/images/normal/history</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="swBarMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pagePath">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btBack">
            <property name="toolTip">
             <string>Back</string>
            </property>
            <property name="statusTip">
             <string>Go to previous directory</string>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
            <property name="icon">
             <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
              <normaloff>:/pktk/images/normal/goback</normaloff>
              <disabledoff>:/pktk/images/disabled/goback</disabledoff>
              <disabledon>:/pktk/images/disabled/goback</disabledon>:/pktk/images/normal/goback</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btUp">
            <property name="toolTip">
             <string>Up</string>
            </property>
            <property name="statusTip">
             <string>Go to parent directory</string>
            </property>
            <property name="text">
             <string>^</string>
            </property>
            <property name="icon">
             <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
              <normaloff>:/pktk/images/normal/goup</normaloff>
              <disabledoff>:/pktk/images/disabled/goup</disabledoff>
              <disabledon>:/pktk/images/disabled/goup</disabledon>:/pktk/images/normal/goup</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BreadcrumbsAddressBar" name="frameBreacrumbPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageQuery">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btBack2">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
              <normaloff>:/pktk/images/normal/goback</normaloff>
              <disabledoff>:/pktk/images/normal/goback</disabledoff>
              <disabledon>:/pktk/images/normal/goback</disabledon>:/pktk/images/normal/goback</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btUp2">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
              <normaloff>:/pktk/images/normal/goup</normaloff>
              <disabledoff>:/pktk/images/disabled/goup</disabledoff>
              <disabledon>:/pktk/images/disabled/goup</disabledon>:/pktk/images/normal/goup</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblViewName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btFilter">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Quick filter</string>
        </property>
        <property name="statusTip">
         <string>Apply a quick filter on current files list</string>
        </property>
        <property name="whatsThis">
         <string>Filter files</string>
        </property>
        <property name="text">
         <string>f</string>
        </property>
        <property name="icon">
         <iconset resource="../../pktk/resources/svg/dark_icons.qrc">
          <normaloff>:/pktk/images/normal/filter_alt</normaloff>
          <disabledoff>:/pktk/images/disabled/filter_alt</disabledoff>:/pktk/images/normal/filter_alt</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetFilter" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="leFilterQuery">
           <property name="statusTip">
            <string>Quick filter: use wildcard * to filter files | To use regular expression, prefix filter with &quot;re:&quot;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BreadcrumbsAddressBar</class>
   <extends>QFrame</extends>
   <header>bulicommander.libs.breadcrumbsaddressbar.breadcrumbsaddressbar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../pktk/resources/svg/dark_icons.qrc"/>
  <include location="../../pktk/resources/svg/light_icons.qrc"/>
 </resources>
 <connections/>
</ui>
